packages/react-components/react-infolabel-preview/stories/InfoLabel/index.stories.tsx

```diff
@@ -0,0 +1 @@
+export {};
```

Comment:
This folder + file was added in order to pass ci, otherwise there's a linting issue preventing ci to pass.

Reply:
hmm this feels like a bug in package generator ? this shouldn't be created only with component generation

Reply:
ok . maybe you added this by some accident  manually ? because our generator won't create this file 

`yarn nx generate @fluentui/workspace-plugin:react-library --name=react-one-two --owner=@microsoft/fluentui-react-build --no-interactive --dry-run `

<img width="940" alt="image" src="https://github.com/microsoft/fluentui/assets/1223799/30a0849d-6deb-4f2a-a7d6-788585bc9d27">